content/news/2021-01-08-mixxx-to-the-first-time-contributor.md

```diff
@@ -0,0 +1,110 @@
+title: Mixxx to the first time contributor
```

Comment:
The title format differs from your other outreachy posts.

Reply:
With the normal format, it becomes "About my Outreachy internship with Mixxx DJ:  Mixxx to the first time contributor". I wanted to maintain the format but I was worried that it was too long....
If it's still okay, then I can change it to that.

Reply:
I think the longer title is okay.